scripts/linters/css_linter.py

```diff
@@ -55,19 +54,7 @@ def _get_trimmed_error_output(css_lint_output):
         Returns:
             str. A string with the trimmed error messages.
         """
-        trimmed_error_messages = []
-        # We need to extract messages from the list and split them line by
-        # line so we can loop through them.
-        css_output_lines = css_lint_output.split('\n')
-        for line in css_output_lines:
-            # Stylelint messages starts with line numbers and then a
-            # "x"(\u2716) and a message-id in the end. We are capturing these
-            # and then replacing them with empty string('').
-            if re.search(r'^\d+:\d+', line.lstrip()):
-                error_message = line.replace(u'\u2716 ', '')
-            else:
-                error_message = line
-            trimmed_error_messages.append(error_message)
+        trimmed_error_messages = css_lint_output.split('\n')
```

Comment:
Considering we are spliting here by `\n` and joining below with the same `\n`, is it needed?

Reply:
Right.